script/Ciel/Day 10/10_01_CIEL10_5_2.txt

```diff
@@ -524,25 +553,26 @@ I mean, in the end―――"
 [sha:cce39ceb66e059cb4d265d40bdd9a023e7689b5f]{
 -- Page 410, Offset 28213.
 -- カタン、と音をたてて畳に黒い影が落ちた。
-A black shape is audibly dropped onto the floor.
+A large black object lands on the tatami with a thud.
 }
 [sha:3159604eb3c443fb1085feec6055879d3a8d162a]{
 -- Page 410, Offset 28214.
 -- 女の両手に収まる程の大きさでありながら、
-Big enough that she has to hold it with both hands―――
+It's so big she has to hold it with both hands.
 }
 [sha:a2208316016344536dc7b3aca7bcb7e21572d816]{
 -- Page 410, Offset 28215.
 -- 刑務所の壁のように<聳|そび>える、
-Towering high like the walls of a prison―――
+It towers above me like the walls of a prison.
 }
 [sha:55c9d0dfd7e6678b165002dc53a786d5d140cae5]{
 -- Page 410, Offset 28216.
 -- 「泣き叫びながら、この中に入るんだから」
-"You'll get inside, still bawling your eyes out."
+// Slight change on the ~ながら, but more canon in EN
+"No matter how much you bawl your eyes out, you're going in here."
 }
 [sha:c0c67a0795a70d6152cc4ae1bf054d51e98b258e]{
 -- Page 410, Offset 28217.
 -- 見覚えのある、正方形の。
-A familiar square shape...
+I've seen that square object before...
```

Comment:
It's much more emphatic in Japanese, and while I'm not sure how to do a similar fragmentary sentence in English, it at least needs to have some oomph since it ends the file. 
>I've seen it before; this square object.